### ISSUE TITLE
Unable to import Forecast from mlforecast

### ISSUE THREAD

**Issue by iki77** [Nixtla/mlforecast#55]
## Description

Unable to import Forecast from mlforecast

## Reproducible example

```python
# code goes here
from mlforecast import Forecast

```

<details>
    <summary>ImportError: cannot import name 'Forecast' from 'mlforecast' (/home/<user>/mambaforge/envs/dev/lib/python3.7/site-packages/mlforecast/__init__.py)</summary>

```python
# Stacktrace
```
</details>

## Environment info
python=3.7
pip installlation mlforecast

Package version: 
mlforecast=0.2.0

**Comment by jmoralez** [comment]
Hi, thanks for raising this. We've made some breaking changes and haven't made a release yet, so most of what you'll see in GitHub and in the docs page isn't available on the PyPI package yet. If you want to try out the new functionalities you can install from github: `pip install git+https://github.com/Nixtla/mlforecast` or you can check the [v0.2.0 tag](https://github.com/Nixtla/mlforecast/tree/v0.2.0) as reference.

**Comment by jmoralez** [comment]
We've released [v0.3.0](https://pypi.org/project/mlforecast/0.3.0/) which should fix this, please let us know if the issue persists.
